core/modules/color/color.module

```diff
@@ -94,6 +94,16 @@ function color_form_system_theme_settings_alter(&$form, &$form_state) {
         ),
       ),
     );
+    $form['color']['custom_name'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Name'),
```

Comment:
Can we please either make the label of this field less vague, or add a `#description` to it?